src/Form/Restaurant/FulfillmentMethodsTrait.php

```diff
@@ -25,7 +26,8 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                 'allow_add' => false,
                 'allow_delete' => false,
                 'prototype' => false,
-            ]);
+            ])
+            ->add('ordersRateLimiter', HiddenType::class);
```

Comment:
This should be available only for users with `ROLE_ADMIN`

Reply:
Oh yeah thanks, good catch :)
[4e08035](https://github.com/coopcycle/coopcycle-web/pull/3584/commits/4e080352f1922625b5007ebe6c9e36d4f2536d54)